examples/nrf24l01_term/nrf24l01_term.c

```diff
@@ -120,8 +123,7 @@ int wireless_cfg(int fd)
   uint32_t rf = DEFAULT_RADIOFREQ;
   int32_t txpow = DEFAULT_TXPOWER;
   nrf24l01_datarate_t datarate = RATE_1Mbps;
-
-  nrf24l01_retrcfg_t retrcfg =
+  struct nrf24l01_retrcfg_t retrcfg =
```

Comment:
`nrf24l01_retrcfg_t` is a typedef (thus the _t) defined in nuttx/wireless/nrf24l01.h file, this is causing the issue.
So `struct` should go away here, even though the coding standard specifies that structs should be typedefed, but this is out of the scope of this PR.

Reply:
I add struct because nxstyle complain this without struct:
```
apps/examples/nrf24l01_term/nrf24l01_term.c:127:2: error: Bad left brace alignment
apps/examples/nrf24l01_term/nrf24l01_term.c:130:2: error: Bad right brace alignment
```

Reply:
Doesn't just need a level of indentation?
```C
  nrf24l01_retrcfg_t retrcfg =
    {
      .count = 5,
      .delay = DELAY_1000us
    };
```

Reply:
Is it right thing to do? struct don't need indent, but typedef need. It's nxstyle's bug which don't understand c grammar.

Reply:
Yes, I agree with you.

Reply:
So I propose that we ignore this warning until nxstyle can do the syntacitc analysis.